jobs-shell/tests/e2e-test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -eux pipefail
+
+# Remove me
+PROJECT_ID=$(gcloud config get project)
+
+gcloud config set run/region "${_REGION}"
+
+# Generate a random 15 character alphanumeric string (lowercase only)
+RAND_ID=$(LC_CTYPE=C tr -dc 'a-z0-9' < /dev/urandom | head -c 15)
+JOB_NAME="jobs-shell-${RAND_ID}"
+
+gcloud builds submit -t "gcr.io/${PROJECT_ID}/jobs-shell"
```

Comment:
Tag with :${SHORT_SHA}

Reply:
Not sure why? Wouldn't source provenance be handled by Container Analysis API if run by trigger?